docs/merged_objects.md

```diff
@@ -225,8 +225,113 @@ Some merged objects may have some additional sample metadata columns specific to
+
+The following additional per-cell data columns for the ADT data can be found in the main `colData` data frame (accessed with `colData(merged_sce)` [as above](#singlecellexperiment-cell-metrics)).
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `altexps_adt_sum`      | UMI count for CITE-seq ADTs                       |
+| `altexps_adt_detected` | Number of ADTs detected per cell (ADT count > 0 ) |
+| `altexps_adt_percent`  | Percent of `total` UMI count from ADT reads       |
+
+
+In addition, the following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), which were calculated on individual libraries before objects were merged, can be found in the `colData` of the `"adt"` alternative experiment, accessed with `colData(altExp(merged_sce, "adt"))`.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs were used |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs were used |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative/isotype control ADTs were not used |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs were not used |
+| `discard`  | Indicates whether the cell should be discarded based on ADT QC statistics |
+
+
+Metrics for each of the ADTs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+rowData(altExp(merged_sce, "adt")) # adt metrics
+```
+
+This data frame contains the following columns with statistics for each ADT.
+The columns `mean-SCPCL000000` and `detected-SCPCL000000` are present for each CITE-seq library in the merged object.
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean-SCPCL000000`      | Mean ADT count across all cells/droplets. Only present for libraries with CITE-seq data                       |
+| `detected-SCPCL000000`  | Percent of cells in which the ADT was detected (ADT count > 0 ). Only present for libraries with CITE-seq data |
+| `target_type` | Whether each ADT is a target (`target`), negative/isotype control (`neg_control`), or positive control (`pos_control`). If this information was not provided, all ADTs will have been considered targets and will be labeled as `target` |
```

Comment:
I just want to note that I like including it in both places!

Reply:
more docs more better ✅